### PR TITLE
Implementation for the HTS wipe operation for fungible and nonfungible tokens

### PR DESCRIPTION
**Pull request by MrValioBg**
**Description**:
Implementation for the HTS wipe operation for fungible and nonfungible tokens

**Related issue(s)**:

Fixes #
#8585 
#8586 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/hedera-app/src/xtest/java/contract/WipeXTest.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                UNAUTHORIZED_SPENDER_ID,
+                Account.newBuilder().accountId(UNAUTHORIZED_SPENDER_ID).build());
+        return accounts;
+    }
+
+    @Override
+    protected void assertExpectedAccounts(@NotNull final ReadableKVState<AccountID, Account> accounts) {
+        final var account = accounts.get(OWNER_ID);
+        assertNotNull(account);
+        assertEquals(NUMBER_OWNED_NFTS - 1, account.numberOwnedNfts());
```
Comment (tinker-michaelj) [inline hedera-node/hedera-app/src/xtest/java/contract/WipeXTest.java:174]: Could add an `assertExpectedTokenRelationships()` that the original balance of 50 fungible units decreased by 20.

**Review by tinker-michaelj — DISMISSED** [review]
LGTM, ty @MrValioBg!

File: hedera-node/hedera-app/src/xtest/java/contract/WipeXTest.java (hunk 1 above)
Comment (MrValioBg) [reply hedera-node/hedera-app/src/xtest/java/contract/WipeXTest.java:179]: Yep, done. Thank you, seems I've missed that :)

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Pull request merged**